### ISSUE TITLE
[🐞] - calling the argocd app argocd

### ISSUE THREAD

**Issue by springdo** [rht-labs/ubiquitous-journey#114]
Looks like on upgrading from v.0.0.5 (which happened automatically btw) causes issues with how we're naming things - specifically the ArgoCD CR. 
Basically, after deploying the operator with the name `argocd` the server pod gets into an infinite loop spinning up and tearing down instances. I reached out to @jmckind in Red Hat and we're able to recreate the issue as follows (create a new ns for each test):  

1) set `startingCSV: argocd-operator.v0.0.5` and `ArgoCD` CR `metadata.name: argocd` 
❌ operator upgrades as expected and recycling termination and creation of `argocd-server` pod
2) set `startingCSV: argocd-operator.v0.0.6` and `ArgoCD` CR `metadata.name: argocd`
❌ no operator upgrade and recycling termination and creation of `argocd-server` pod
3) set `startingCSV: argocd-operator.v0.0.5` and `ArgoCD` CR `metadata.name: my-argocd` 
✅ operator upgrate and solid single `argocd-server` pod
4) set `startingCSV: argocd-operator.v0.0.6` and `ArgoCD` CR `metadata.name: my-argocd`
✅ no  operator upgrade and solid single `argocd-server` pod

As a work around we can over write the name, but we should keep an eye out for a fix. My GoLang is probs not up to snuff to fix upstream....

**Comment by eformat** [comment]
ah yup @springdo - i too found this

https://github.com/rht-labs/ubiquitous-journey/pull/116

**Comment by springdo** [comment]
Yeah we can keep an eye on things in the operator but it's probs connected to how it does the deploy according to the guy who wrote it. I guess we should move this issue upstream to the operator? I'll merge your pr for now as a work around anyways. I thought I'd actually already made that change and it was merged but now discovered i never added the name change on my pr after I made it a variable 🤦‍♂️
